Configuration/ProcessModifiers/python/runDependentForPixelPUMix_cff.py

```diff
@@ -0,0 +1,2 @@
+import FWCore.ParameterSet.Config as cms
+runDependentForPixelPUMix =  cms.Modifier()
```

Comment:
hi @carolinecollard - apologies, haven't read yet the message you sent be e-mail, but why do we need two process modifiers?

Reply:
you were reacting to the commit before I post the explanation :) --> see below...